file-echo-api/src/FileEchoServer.hs

```diff
@@ -2,14 +2,16 @@
 module FileEchoServer ( module FileEchoServer ) where
 
 import qualified Argo as Argo
-import           Control.Monad.IO.Class ( liftIO )
+import Argo.Doc
```

Comment:
`import Argo.Doc` => `import qualified Argo.Doc as Doc` or similar IMO would be better. (Since part of the purpose of this module is to illustrate how to use Argo, erring on the side of verbose/pedantic imports I think might help some quickly grok things).

Reply:
Sounds good. I'll make that change!

Reply:
Done.